### PR TITLE
Adding common schedules

### LINKED ISSUE #138: Add common schedules
From what I've heard people usually teach either the data organization, the unix shell, and data wrangling OR data organization, the unix shell, and the R lesson as both options are about 2 days/4 half days of materials.  I've met with a few instructors teaching this for the first time and they were surprised to hear they couldn't teach all 4 lessons in 2 days/4 half days.  Perhaps we should add a few common schedules to the front page.

1:  data organization, the unix shell, and data wrangling (2 days/ 4 half days)
2: data organization, the unix shell, and the R lesson (2 days/ 4 half days)
2: data organization, the unix shell, data wrangling, and the R lesson (3 days / 6 half days)

### PR DESCRIPTION
**Pull request by sstevens2**
Fixes #138

Noticed the heading levels when I went to edit this.  I left it as is to not make multiple changes in one PR but I think these should be level 2 headings.  My understanding is that there should only be one level 1 heading and then the other page headings should be hierarchical under it.

[Heading Accessibility Reference](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/)

### CODE REVIEW

**Comment by sstevens2** [comment]
I did make the subheadings in this section level 3 so they won't need to be edited if we switch the section headings to 2.

File: index.md [hunk 1]
```diff
@@ -58,3 +58,29 @@ This workshop is designed to be run on pre-imaged Amazon Web Services (AWS)
 instances. All the software and data used in the workshop are hosted on an Amazon Machine Image (AMI).
 If you want to run your own instance of the server used for this workshop, follow the directions in the [Setup](setup.html) tab. 
 
+
+# Common Schedules
+
+### Schedule A(2 days OR 4 half days)
```
Comment (sstevens2) [inline index.md:64]:
```suggestion
### Schedule A (2 days OR 4 half days)
```

**Review by JCSzamosi — CHANGES_REQUESTED** [review]
It's a small thing, but can you make the spacing around the ampersand consistent? Right now you have "Half-day 3 &4:" twice, and then "Half-day 5&6:"

My preference would be for spaces before and after all ampersands, but I don't know if it matters once rendered.

**Comment by JCSzamosi** [comment]
@twrightsman, I noticed you added this PR to the Workbench Transition milestone, but I don't think that's correct. Can you explain why you did that?

File: index.md [hunk 2]
```diff
@@ -58,3 +58,29 @@ This workshop is designed to be run on pre-imaged Amazon Web Services (AWS)
...
 If you want to run your own instance of the server used for this workshop, follow the directions in the [Setup](setup.html) tab. 
 
+
+# Common Schedules
+
+### Schedule A (2 days OR 4 half days)
+
+Half-day 1: [Project organization and management](https://datacarpentry.github.io/organization-genomics/) & [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/)
+Half-day 2: [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/) (continued)
+Half-day 3 &4 : [Data wrangling and processing](https://datacarpentry.github.io/wrangling-genomics/)
```
Comment (sstevens2) [inline index.md:68]:
```suggestion
Half-day 3 & 4 : [Data wrangling and processing](https://datacarpentry.github.io/wrangling-genomics/)
```

File: index.md [hunk 3]
```diff
@@ -58,3 +58,29 @@ This workshop is designed to be run on pre-imaged Amazon Web Services (AWS)
...
+
+Half-day 1: [Project organization and management](https://datacarpentry.github.io/organization-genomics/) & [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/)
+Half-day 2: [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/) (continued)
+Half-day 3 &4: [Intro to R and RStudio for Genomics](https://datacarpentry.org/genomics-r-intro/)
+
+### Schedule C (3 days OR 6 half days)
+
+Half-day 1: [Project organization and management](https://datacarpentry.github.io/organization-genomics/) & [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/)
+Half-day 2: [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/) (continued)
+Half-day 3 &4 : [Data wrangling and processing](https://datacarpentry.github.io/wrangling-genomics/)
```
Comment (sstevens2) [inline index.md:80]:
```suggestion
Half-day 3 & 4 : [Data wrangling and processing](https://datacarpentry.github.io/wrangling-genomics/)
```

File: index.md [hunk 4]
```diff
@@ -58,3 +58,29 @@ This workshop is designed to be run on pre-imaged Amazon Web Services (AWS)
...
+Half-day 5&6: [Intro to R and RStudio for Genomics](https://datacarpentry.org/genomics-r-intro/)
```
Comment (sstevens2) [inline index.md:81]:
```suggestion
Half-day 5 & 6: [Intro to R and RStudio for Genomics](https://datacarpentry.org/genomics-r-intro/)
```

**Comment by sstevens2** [comment]
Nice catch @JCSzamosi!  I put in a commit with your change to spaces before and after as I agree that probably looks better.

> It's a small thing, but can you make the spacing around the ampersand consistent? Right now you have "Half-day 3 &4:" twice, and then "Half-day 5&6:"
> 
> My preference would be for spaces before and after all ampersands, but I don't know if it matters once rendered.
> 
> EDIT: I just realized I probably should have tagged @sstevens2 in this request.

File: index.md [hunk 5]
```diff
@@ -58,3 +58,29 @@ This workshop is designed to be run on pre-imaged Amazon Web Services (AWS)
...
+
+Half-day 1: [Project organization and management](https://datacarpentry.github.io/organization-genomics/) & [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/)
+Half-day 2: [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/) (continued)
+Half-day 3 & 4 : [Data wrangling and processing](https://datacarpentry.github.io/wrangling-genomics/)
+
+### Schedule B (2 days OR 4 half days)
+
+Half-day 1: [Project organization and management](https://datacarpentry.github.io/organization-genomics/) & [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/)
+Half-day 2: [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/) (continued)
+Half-day 3 &4: [Intro to R and RStudio for Genomics](https://datacarpentry.org/genomics-r-intro/)
```
Comment (sstevens2) [inline index.md:74]:
```suggestion
Half-day 3 & 4: [Intro to R and RStudio for Genomics](https://datacarpentry.org/genomics-r-intro/)
```

**Comment by twrightsman** [comment]
> @twrightsman, I noticed you added this PR to the Workbench Transition milestone, but I don't think that's correct. Can you explain why you did that?

Sure thing. Any fork of the repository that was created before the Workbench transition will be invalid after the repository is transitioned to Workbench around September. So all PRs need to be resolved before the pre-transition milestone is due (September). It would probably be better named as "Workbench Transition Preparation"?

**Comment by JCSzamosi** [comment]
> > @twrightsman, I noticed you added this PR to the Workbench Transition milestone, but I don't think that's correct. Can you explain why you did that?
> 
> Sure thing. Any fork of the repository that was created before the Workbench transition will be invalid after the repository is transitioned to Workbench around September. So all PRs need to be resolved before the pre-transition milestone is due (September). It would probably be better named as "Workbench Transition Preparation"?

Ohhh. I was confused. The genomics-organization repo has already transitioned to workbench, but now I see this one has not. I understand now.

**Review by JCSzamosi — APPROVED** [review]

**Comment by JCSzamosi** [comment]
> Nice catch @JCSzamosi! I put in a commit with your change to spaces before and after as I agree that probably looks better.
> 

This looks good to me! I'm going to merge the pull request now.

**Pull request merged**